README.md

```diff
@@ -143,9 +144,18 @@ export RESOURCE_GROUP_NAME=give_your_name_citus_test_automation_r_g
 cd azure
 
 # open and modify the instance types/discs as you wish
+#
+# Note: For valgrind tests, setting `numberOfWorkers` to a value is useless as it will be overwritten 
```

Comment:
I'd rather move this and the below comments to  somewhere else such as `Running Valgrind Tests` as this point is a generic place and, anyone interested in valgrind  test should read the `Running Valgrind Tests` section already

Reply:
done